### PR TITLE
batches: add basic library pane for SSBC editor

### PR DESCRIPTION
**Pull request by courier-new**
Adds back the minimal example specs to choose from when writing a new batch spec, and a confirmation modal so that you don't accidentally erase your editor contents if you select one by accident. 😜

Ignore the drawer UI... Our version of reactstrap's `Collapse` only does up/down collapsing, not left/right, and trying to upgrade the package breaks our tooltips and a handful of other things. 😐 But that's not important for now!

### CODE REVIEW

File: client/web/src/enterprise/batches/create/NewCreateBatchChangePage.module.scss [hunk 1]
```diff
@@ -19,3 +19,8 @@
     border: none;
     width: 30%;
 }
+
+.storybook-container {
```
Comment (eseliger) [inline client/web/src/enterprise/batches/create/NewCreateBatchChangePage.module.scss:23]: It seems a bit strange to me to include storybook only styles in the main module, maybe create a separate one for the story at `NewCreateBatchChangePage.story.module.scss`?

File: client/web/src/enterprise/batches/create/library/LibraryPane.tsx [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+                    >
+                        {collapsed ? (
+                            <ChevronDoubleRightIcon className="icon-inline mr-1" />
+                        ) : (
+                            <ChevronDoubleLeftIcon className="icon-inline mr-1" />
+                        )}
+                    </button>
+                </div>
+
+                {/* TODO: This should slide vertically but not on our version of reactstrap. */}
```
Comment (eseliger) [inline client/web/src/enterprise/batches/create/library/LibraryPane.tsx:66]: Damn you reactstrap :D

**Review by eseliger — APPROVED** [review]

File: client/web/src/enterprise/batches/create/NewCreateBatchChangePage.module.scss (hunk 1 above)
Comment (courier-new) [reply client/web/src/enterprise/batches/create/NewCreateBatchChangePage.module.scss:23]: Yeah totally fair. I'm not even reusing this anywhere so I think I'll just disable the eslint rule and define it inline.

File: client/web/src/enterprise/batches/create/library/LibraryPane.tsx (hunk 2 above)
Comment (courier-new) [reply client/web/src/enterprise/batches/create/library/LibraryPane.tsx:66]: I knowwww 😭 I tried to fix it when I upgraded the version but there were too many things that were wrong or weird for a half-day timebox lol. In the same half-day timebox I probably could have just implemented something with CSS, sigh.

Comment (eseliger) [reply client/web/src/enterprise/batches/create/library/LibraryPane.tsx:66]: 😢 🤷

**Pull request merged**